resident-default.properties

```diff
@@ -796,9 +796,9 @@ mosip.resident.download.card.naming.convention=Get_my_UIN_{timestamp}
 
 # Credential request config for sharing credential to partner
 mosip.resident.request.credential.credentialType=vercred
-#mosip.resident.request.credential.credentialType=euin
-#mosip.resident.request.credential.isEncrypt=true
-#mosip.resident.request.credential.encryption.key=abc123
+mosip.resident.request.credential.credentialType=euin
+mosip.resident.request.credential.isEncrypt=true
+mosip.resident.request.credential.encryption.key=abc123
```

Comment:
Encryption.key is hard-coded 

Please ensure to use as a placeholder & randomly generated key

Please check with developer

Reply:
updated .